exercises/es/exc_03_07.py

```diff
@@ -2,8 +2,8 @@
 from spacy.matcher import PhraseMatcher
 from spacy.tokens import Span
 
-nlp = spacy.load("en_core_web_sm")
-animals = ["Golden Retriever", "cat", "turtle", "Rattus norvegicus"]
+nlp = spacy.load("es_core_news_sm")
+animals = ["Golden Retriever", "gato", "tortuga", "rata"]
```

Comment:
Maybe we can have one more example of a multi-word phrase pattern here? Maybe an animal with two or three tokens and all lowercase or a combination of lowercase/capitalised? Just to show what's possible.

Reply:
Done! Added a really nice Colombian animal: "oso de anteojos" -> bear with spectacles